gemfiles/activerecord_5.1.gemfile

```diff
@@ -11,6 +11,7 @@ when "mysql"
 else
   gem 'sqlite3'
 end
+gem 'acts_as_paranoid'
```

Comment:
Why not remove acts_as_paranoid dependency completely from Gemfiles, and instead check in code if `deleted_at` column exists or not

Reply:
Okay will find a way. Thanks @brissenden